bugsnag-android-core/src/main/java/com/bugsnag/android/FileStore.java

```diff
@@ -83,8 +83,18 @@ void enqueueContentForDelivery(String content) {
             out = new BufferedWriter(new OutputStreamWriter(fos, "UTF-8"));
             out.write(content);
         } catch (Exception exc) {
+            File errorFile = new File(filename);
+
             if (delegate != null) {
-                delegate.onErrorIOFailure(exc, new File(filename), "NDK Crash report copy");
+                delegate.onErrorIOFailure(exc, errorFile, "NDK Crash report copy");
+            }
+
+            try {
+                if (!errorFile.delete()) {
+                    errorFile.deleteOnExit();
+                }
+            } catch (Exception ex) {
+                Logger.warn("Failed to delete partially written file", ex);
```

Comment:
This functional change isn't mentioned in the PR description nor a changelog entry.

Reply:
This was mentioned at the start of the changeset in the PR description - I'll add a changelog entry and rethink how the change has been worded.

Reply:
FWIW, can you rename the PR and add the behaviour change to the Goal as it's easy to miss?